src/View/Helper/FormHelper.php

```diff
@@ -1044,6 +1047,19 @@ public function control(string $fieldName, array $options = []): string
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (
+            !empty($options['label']) && $options['type'] != 'hidden' ||
+            !empty($options['placeholder']) && $options['type'] != 'hidden'
+        ) {
+            $isFieldError = $this->isFieldError($fieldName);
+            $options += [
+                'aria-label' => !empty($options['label']) ? $options['label'] : $options['placeholder'],
```

Comment:
The `label` option can also be an array. We'll need to work around that.
```suggestion
                'aria-label' => isset($options['label']) && is_string($options['label']) ? $options['label'] : $options['placeholder'],
```